Added mention of updated Boostrap (#2344)
ibexa/documentation-developer: docs/update_and_migration/from_3.3/to_4.0.md

## Update your custom code

> ### Back Office customizations

Reviewer: ```suggestion
### Back Office customization
```

> The v4 version of [[= product_name =]] is using Bootstrap 5 in the Back Office. If you've used Bootstrap 4 to style your custom Back Office components, update them [using the migration guide from Bootstrap 4](https://getbootstrap.com/docs/5.0/migration/).

Reviewer: ```suggestion
The v4 version of [[= product_name =]] is using Bootstrap 5 in the Back Office. If you were using Bootstrap 4 for styling, you need to update and adjust all custom Back Office components [following the migration guide from Bootstrap 4](https://getbootstrap.com/docs/5.0/migration/).
```

> The v4 version of [[= product_name =]] is using Bootstrap 5 in the Back Office. If you've used Bootstrap 4 to style your custom Back Office components, rewrite them [using the migration guide from Bootstrap 4](https://getbootstrap.com/docs/5.0/migration/).

Reviewer 2: IMHO "migrate to" / "upgrade to" sound better then "rewrite"
